src/Common/tests/System/PlatformDetection.cs

```diff
@@ -18,9 +18,10 @@ public static partial class PlatformDetection
         // do it in a way that failures don't cascade.
         //
 
-        public static bool IsUap => IsWinRT || IsNetNative;
+        public static bool IsUap => IsInAppContainer || IsNetNative;
```

Comment:
I am seeing uap and uapaot 2 different frameworks. 

@ViktorHofer you may define the definition matrix for possible frameworks support (and may be with combination with running environment) something like:

- UAP (equivalent to F5 UWP, which is IsInAppContainer  and not native
- UAPOAT is native (can run on Linux)
- UAPOAT with WinRT (retail scenario of UWP) which is IsInAppContainer and native
- UAP with no IsInAppContainer. I don't think we need this at all.